src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -14,7 +14,7 @@
     <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
     <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">21.0</SupportedOSPlatformVersion>
     <SupportedOSPlatformVersion Condition="$(TargetFramework.Contains('-windows'))">10.0.17763.0</SupportedOSPlatformVersion>
```

Comment:
should we update this as well?

Reply:
For sure. 

Let's use `windows10.0.19041` to match the .NET MAUI Library templates